<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_datainput</class>
 <widget class="QDialog" name="Dialog_datainput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>898</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>236</red>
        <green>236</green>
        <blue>236</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: #FFE2D7;
    border: 2px solid #FF7740;
    border-radius: 12px;  /* ← 더 둥글게 */
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #FFCBB6;  /* 살짝 짙은 살구색으로 호버 효과 */
}

QListWidget {
    border: 2px solid #FF7740;  /* 오렌지색 테두리 */
    border-radius: 12px;        /* 둥근 모서리 */
    padding: 4px;
    background-color: white;    /* 배경 흰색 (선택) */
}

QComboBox {
    border: 2px solid #FF7740;
    border-radius: 12px;
    padding: 4px 30px 4px 10px; /* 왼쪽 10px, 오른쪽 30px */
    background-color: white;
    font-size: 14px;
}

/* 🔧 드롭다운 영역 스타일 설정 – 화살표가 안 보이거나 눌리지 않는 문제 방지 */
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border: none;
    padding-right: 4px;
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton_uploaddata">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>800</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Upload Data</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submitinput">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>740</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Submit Input</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_savedata">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>800</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_position">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>570</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_timeline">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>620</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time Line</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_datalinput">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>670</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Data Input</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_position">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>570</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_timeline">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>620</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_datainput">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>670</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QGraphicsView" name="footballfield">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>90</y>
     <width>600</width>
     <height>386</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>290</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>420</width>
     <height>290</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGraphicsView" name="logo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>420</width>
     <height>290</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>661</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_datalog">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Data Log</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_eventmap">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Event Map</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="statguide">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>661</width>
     <height>401</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>290</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>420</width>
     <height>290</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QGraphicsView#statguide {
    border: 2px solid #FF7740;  /* 오렌지색 테두리 */
    border-radius: 12px;        /* 둥근 모서리 */
    padding: 4px;
    background-color: white;    /* 배경 흰색 (선택) */
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_statguide">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stat Guide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>110</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_export">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_uploadvideo">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>740</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Upload video</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_mode">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_1sthalf">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>500</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1st Half</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_plus">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>620</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_minus">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>620</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2ndhalf">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>530</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2nd Half</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_home">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_away">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>530</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Away</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_left">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>530</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;&lt;&lt;&lt;&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_right">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>530</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;&gt;&gt;&gt;&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_half">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>510</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Half</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_team">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>510</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Team</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_attackdirection">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>501</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Attack Direction</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
